src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration-VS2017.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="NLog" Version="4.4.5" />
+    <PackageReference Include="NLog" Version="4.4.0-beta13" />
```

Comment:
Oh yer, forget the NLog guys did that, had to remove .NET Core support for NLog in #228 but didn't actually change project.json. We shipped with the nuspec supporting 4.4.1, can we just target .NET 4.5?